client/src/pages/Discover/ContestCard/ContestCard.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import useStyles from './useStyles';
+import moment from 'moment';
+import { Typography, Grid, Card, CardActionArea, CardMedia, CardContent } from '@material-ui/core';
+
+interface Props {
+  title: string;
+  description: string;
+  prizeAmount: number;
+  image: string;
+  deadlineDate: string;
+  id: string;
+}
+
+export default function ContestCard({ title, description, prizeAmount, image, deadlineDate, id }: Props): JSX.Element {
+  const classes = useStyles();
+
+  return (
+    <Card elevation={2}>
+      <CardActionArea component={Link} to={'/contest/' + id}>
+        <CardMedia>
```

Comment:
The Card Media component takes an image as a prop, maybe consider using it?

Ref: https://material-ui.com/components/cards/

Reply:
Successfully implemented at commit 03e97656998da9be9788a9e83a0d734c45c19266